intera_interface/src/intera_motion_interface/interaction_options.py

```diff
@@ -245,27 +245,21 @@ def set_interaction_control_mode(self, interaction_mode = None):
                 rospy.logerr('The number of interaction_control_mode elements must be 1 or %d',
                              self.n_dim_cart)
 
-    def set_disable_damping_in_force_control(self, disable_damping_in_force_control = None):
+    def set_disable_damping_in_force_control(self, disable_damping_in_force_control = self.default_disable_damping_in_force_control):
```

Comment:
Oops typo I have self in there.

Reply:
I thought you would need the `self.` - do you not?